scripts/globals/regimes.lua

```diff
@@ -1146,7 +1146,7 @@ dsp.regime.bookOnEventFinish = function(player, option, regimeType)
             else
                 power = 56
             end
-
+            power = ( power * 100 ) / 256
```

Comment:
Current shell spell scripts use whole numbers for the power. Copy those.

E.G. Shell = 9, Shell II = 14, etc.